Views/Home/Index.cshtml

```diff
@@ -3,6 +3,6 @@
 }
 
 <div class="text-center">
-    <h1 class="display-4">Welcome</h1>
+    <h1 class="display-4">Hello IT-Team</h1>
```

Comment:
Please add a "!" after IT-Team to make it better

Reply:
Mach ich gleich.